### PR TITLE
user identification was added

### PR DESCRIPTION
**Pull request by Victoria-Moiseienko**

### CODE REVIEW

File: src/main/java/com/internet/shop/controllers/carts/AddProductToCartController.java [hunk 1]
```diff
@@ -23,8 +24,11 @@
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
+        HttpSession session = req.getSession();
+        Long userId = (Long) session.getAttribute(USER_ID);
+
```
Comment (MaliukDaria) [inline src/main/java/com/internet/shop/controllers/carts/AddProductToCartController.java:29]: empty line?

File: src/main/java/com/internet/shop/controllers/carts/AddProductToCartController.java [hunk 2]
```diff
@@ -23,8 +24,11 @@
...
         Long productId = Long.valueOf(req.getParameter("productId"));
-        ShoppingCart shoppingCart = shoppingCartService.getByUserId(USER_ID);
+        ShoppingCart shoppingCart = shoppingCartService.getByUserId(userId);
         shoppingCartService.addProduct(shoppingCart, productService.get(productId));
```
Comment (MaliukDaria) [inline src/main/java/com/internet/shop/controllers/carts/AddProductToCartController.java:33]: empty line

File: src/main/java/com/internet/shop/controllers/carts/DeleteProductFromCartController.java [hunk 3]
```diff
@@ -21,9 +22,12 @@
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
+        HttpSession session = req.getSession();
+        Long userId = (Long) session.getAttribute(USER_ID);
+
```
Comment (MaliukDaria) [inline src/main/java/com/internet/shop/controllers/carts/DeleteProductFromCartController.java:27]: empty line

File: src/main/java/com/internet/shop/controllers/carts/GetCartProductsController.java [hunk 4]
```diff
@@ -19,7 +20,10 @@
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        ShoppingCart shoppingCart = shoppingCartService.getByUserId(USER_ID);
+        HttpSession session = req.getSession();
+        Long userId = (Long) session.getAttribute(USER_ID);
+
```
Comment (MaliukDaria) [inline src/main/java/com/internet/shop/controllers/carts/GetCartProductsController.java:25]: empty line

**Review by KseniiaMakarova — APPROVED** [review]
I agree with Daria, too many empty lines usually make code less readable

**Pull request merged**